ui/base-tool/src/main/res/drawable/ic_settings.xml

```diff
@@ -0,0 +1,18 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
this image should have been moved from the `tract-renderer` module to the `base-tool` module